lang/llvm-3.9/Portfile

```
@@ -57,8 +57,9 @@ if {${subport} eq "llvm-${llvm_version}"} {
 
     # Older Xcode's lipo doesn't support x86_64h slices
     # https://trac.macports.org/ticket/53159#ticket
+    # change to depends_lib so cctools is also available at runtime
```

I would rewrite this comment to explain why we want it available at runtime:
```suggestion
    # Older Xcode's ranlib doesn't understand objects produced by newer clang (malformed object (unknown load command 2))
    # https://trac.macports.org/ticket/57412
```

You could also remove the unnecessary `#ticket` from the preceding ticket URL.